src/components/editor/Editor.jsx

```diff
@@ -69,9 +79,15 @@ class Editor extends Component {
             <h1>[Clone|Edit] <em>Name of Resource</em></h1>
           </section>
           <section className="col-md-3">
-            <button type="button" className="btn btn-primary btn-sm">Preview RDF</button>
+            <button type="button" className="btn btn-primary btn-sm" onClick={this.handleRdfShow}>Preview RDF</button>
           </section>
         </div>
+        <div>
+           <RDFModal show={this.state.showRdf}
+                     close={this.handleRdfClose}
+                     rtId={this.props.rtId}
+                     linkedData={ JSON.stringify(this.props.rdf) }/>
```

Comment:
Instead of the Editor React component having this RDF prop, we should create a new `mapDispatchToProps` function in the RDFModal class that generates the RDF based on the redux state.